subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy

```diff
@@ -33,20 +33,20 @@ class CheckstylePluginIntegrationTest extends WellBehavedPluginTest {
      */
     def setup() {
         buildFile << """
-            apply plugin: 'groovy'
+            apply plugin: 'java'
+            apply plugin: 'checkstyle'
```

Comment:
Note that the inherited test cases all apply the plugin under test automatically (some, qualified/unqualified, some only to an included build). 

Isn't this adding a second application? Is that intentional? I would leave the explicit application to the one test case in this class.

Also, I don't get the switch from groovy to java.

❓

Reply:
I'll remove the explicit `checkstyle` application then.

My thought was that groovy plugin applies the java plugin, which is all that is needed to demonstrate the behavior, so why use groovy.

Reply:
I will defer to you on that one, Tom, seems reasonable to me